bin/incoming-mail-inject.pl

```diff
@@ -40,55 +41,46 @@ ()
         768 * 1024;
 }
 
-my $buf;
 my $msg = '';  # in-memory message
-my $rv;
-my $len = 0;
-my $ieh;
-my $ignore_message = 0;  # bool: to ignore rest of message.
+my $len = 0;   # length of message
+
 eval {
-    while ($rv = sysread(STDIN, $buf, 1024*64)) {
+    my ( $buf, $rv );
+    while ( $rv = sysread STDIN, $buf, 1024*64 ) {
         next if $ignore_message;
```

Comment:
This fails to compile - `Global symbol "$ignore_message" requires explicit package name`

I think you've streamlined the process such that this line should just be removed?